content/en/news/releases/1.4.x/announcing-1.4.3/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Istio 1.4.3
+linktitle: 1.4.3
+subtitle: Patch Release
+description: Istio 1.4.3 patch release.
+publishdate: 2020-01-08
+release: 1.4.3
+aliases:
+    - /news/announcing-1.4.3
+---
+
+This release includes bug fixes to improve robustness. This release note describes what’s different between Istio 1.4.2 and Istio 1.4.3.
+
+{{< relnote >}}
+
+## Bug fixes
+
+- **Fixed** an issues where Mixer creates too many watches, overloading the apiserver ([Issue 19481](https://github.com/istio/istio/issues/19481)).
```

Comment:
@kyessenov we could make this more strongly worded, what do you think? Maybe give it a standalone section?

Reply:
It was a bug fix for a slow leak. Putting it first in this list seems fair. Also, please fix the grammar ("an _issue_").